ccd-definition/CaseField.json

```diff
@@ -279,5 +279,13 @@
     "FieldType": "FixedRadioList",
     "FieldTypeParameter": "ResponseIntention",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "legacyReferenceNumber",
```

Comment:
Can we call this legacyCaseReference, please?

Reply:
renamed as advised.